Add #hs-api pages

alexsoble opened the pull request:
## What does this change? 

Add a v1 draft of the Eligibility APIs Initiative project site. 

## Federalist preview link

+ [Background](https://federalist-65bb532b-030a-4b62-a416-d7b74e4c0f2a.app.cloud.gov/preview/18f/portfolios/ars/hs-apis-subsite/projects/eligibility-apis/)

AshleyO85 commented:
procurement language looks good!

alexsoble commented:
Thanks @AshleyO85 ! 😄 😄

stvnrlly commented on _layouts/hs-apis-page.html:
```diff
@@ -0,0 +1,54 @@
+    </div>
+  </section>
+
+  <section class="grid-container usa-section content-wide">
```
Would it make sense to give this some kind of element on top to make it clear that it's the side nav? I was thrown for a minute by seeing "background" and "background" next to each other before I figured it out.

alexsoble replied:
@stvnrlly Thanks for catching this, added code comments to make this clearer!

alexsoble commented:
@Jacklynn Made the following assets changes that we discussed in standup: 

+ Added logo to header
+ Added gavel icon
+ Use static .png calculator image for now 

Let me know if the assets look in good shape for a v1 🚢 ! If we have any extra hours we can work on improvements like using the calculator GIF with a link to disable animations, and potentially adding more illustrations.

Jacklynn commented:
@alexsoble I'd leave the alt tag empty since it's a decorative image `alt=""`

The h2 creates an extra space throwing the graphic and title proximity off 😔 
I'd either
1. Reduce the margin above the H2, this is at 8px (ideal, but the most custom)
<img width="563" alt="Screen Shot 2020-09-15 at 11 38 20 AM" src="https://user-images.githubusercontent.com/2560618/93250765-23e2e600-f748-11ea-983c-667f37aee830.png">
2. Put the graphic under the H2, There's slightly less margin after the title so I think this ok
<img width="600" alt="Screen Shot 2020-09-15 at 11 38 53 AM" src="https://user-images.githubusercontent.com/2560618/93250844-3eb55a80-f748-11ea-9153-41a499a888ee.png">
3. remove the graphic

Thank you so much!

alexsoble commented:
@Jacklynn Removed alt tag and moved image below h2 in ff82b065f57c960ccd5a5ad631dcc29d5926a30a!

Jacklynn commented:
@alexsoble I don't know if it's just my dev tools, but it looks like it just ends at `alt` and not the full `alt=""` -- does it do that for you or make a difference? I wonder if it looks less intentionally blank without the full attribute?

alexsoble commented:
@Jacklynn — let me know what you think of the latest. We can keep improving this but I don't think it necessarily needs to be a blocker to 🚢  ing today

Jacklynn commented:
looks good to me, thanks!
